src/main/java/org/wahlzeit_revisited/utils/WahlzeitConfig.java

```diff
@@ -1,3 +1,23 @@
+/*
+ * Copyright (c) 2006-2009 by Dirk Riehle, http://dirkriehle.com, Aron Metzig
```

Comment:
I think the correct way to declare multiple authors according to https://reuse.software/faq/ is:
```java
/*
 * Copyright (c) 2006-2009 by Dirk Riehle, http://dirkriehle.com
 * Copyright (c) 2021 by Aron Metzig
 * ....
*/
``

Reply:
fixed in a7fe0cd